qcodes/instrument/group_parameter.py

```diff
@@ -79,9 +77,9 @@ class Group:
     via the same command. The command has to be a string, for example,
     a VISA command.
 
-    The `Group`'s methods are used within `GroupParameter` in order to
-    properly implement setting and getting of a single parameter in the
-    situation where one command sets or gets more than one parameter.
+    The :meth:`Group`'s methods are used within :class:`GroupParameter` in
```

Comment:
this is about "methods of Group class" (not about a method called "Group").
```suggestion
    The :class:`Group`'s methods are used within :class:`GroupParameter` in
```

Reply:
Corrected this.